### PR TITLE
Postgres fs impl -- WIP

### PR DESCRIPTION
**Pull request by Gastove**
Opening this PR for general eyes-on-the-project purposes -- this once again switches a decent amount of stuff around. Mostly so far: swaps bidi for compojure.

### CODE REVIEW

File: src/doctopus/db/schema.clj [hunk 1]
```diff
@@ -87,8 +89,20 @@
 (defn- create-fts-document-index
   [db-name]
   (let [update-sql "UPDATE documents SET search_vector = to_tsvector('english', name || ' ' || body)"
-        idx-sql "CREATE INDEX fts_idx ON documents USING GIN(search_vector)"]
-    (log/info "Creating FTS Index for the documents table")
+        idx-sql (->> ["DO $$"
```
Comment (jarofghosts) [inline src/doctopus/db/schema.clj:92]: not to tack on more deps, but wondering if you have looked at [yesql](https://github.com/krisajenkins/yesql)

Comment (Gastove) [inline src/doctopus/db/schema.clj:92]: Ohhh, that is very pleasant! Hadn't met that one before. Might just put that on my to-implement list.

Comment (Gastove) [inline src/doctopus/db/schema.clj:92]: I think moving all the raw SQL in to yesql sounds like a lovely improvement. For now, everywhere we _are_ using Korma is veeeeeery simple and inoffensive, so I'm inclined to leave it in place. (Besides, despite yasql's author's grousing, Korma is _pretty nice_ for little things.)

**Comment by fardog** [comment]
preliminary review says :+1: (please note: i merged in master to do the test against the latest js stuff, and i only tested the fs impl to ensure nothing broke, did not test postgres since it's not ready)

**Comment by Gastove** [comment]
This should be basically good to go! 

Closes #51 
Closes #63

**Comment by fardog** [comment]
:+1:

**Pull request merged**